packages/sdk/src/client.ts

```diff
@@ -299,14 +310,21 @@ export function SoundClient({
           })
         }
 
-        return merkleDropMinter.mint(
+        const mintArgs = [
           mintSchedule.editionAddress,
           mintSchedule.mintId,
           quantity,
           proof,
           affiliate,
           txnOverrides,
-        )
+        ] as const
+
+        const gasEstimate = await merkleDropMinter.estimateGas.mint(...mintArgs)
+
+        // Add a buffer to the gas estimate to account for node provider estimate variance due to edition mintRandomness calculation
+        txnOverrides.gasLimit = gasLimit || gasEstimate.add(MINT_GAS_LIMIT_BUFFER)
```

Comment:
Instead of a fixed buffer, make it variable, multiply the estimate by 1.2 for a 20% increase is something I’ve seen commonly